extensions/email/src/test/resources/retriever/imap-flows.xml

```diff
@@ -73,9 +73,20 @@
     </flow>
 
     <flow name="retrieveAndDelete">
+        <email:list-imap config-ref="config" deleteAfterRetrieve="true"/>
```

Comment:
is `deleteAfterRetrieve` the exact same parameter name that the `list-pop` operation has?